components/LabelForm.js

```diff
@@ -5,13 +5,14 @@ import { CheckBox } from '@rneui/themed'
 
 import { COLORS } from '../theme/colors'
 
+import { FormLabel } from './FormLabel'
 import { styles } from './LabelForm.styles'
 import { TextSourceSans } from './TextSourceSans'
 
 export const LabelListContainer = (props) => {
   return (
     <View style={styles.labelContainer}>
-      <TextSourceSans style={styles.formLabel}>{props.field}</TextSourceSans>
+      <FormLabel>{props.field}</FormLabel>
       {props.children}
       {props.children.touched &&
         <TextSourceSans style={styles.formError} accessibilityLiveRegion="polite">{props.error}</TextSourceSans>
```

Comment:
would it make sense to have an equivalent component for  form errors? for instance `<FormError>` ?

Reply:
@khamui yes! should I add it in this PR do you think? was also thinking containers too?

Reply:
ok added just the form error for now